vendor/github.com/kermieisinthehouse/systray/systray_nonwindows.go

```diff
@@ -1,5 +1,5 @@
-//go:build !windows
-// +build !windows
+//go:build !windows !freebsd
```

Comment:
thats probably where the error is from
didnt notice that the cross compile was using the vendor file
should be
`//go:build !windows || !freebsd`
at least for the syntax